doc/batch_settings.rst

```diff
@@ -0,0 +1,110 @@
+**************
+
+========
+Overview
+========
+SmartSim offers functionality to launch workloads (``Model`` or ``Ensemble``)
+as batch jobs. Each SmartSim `launcher` interfaces with a ``BatchSettings`` object
+specific to a systems Workload Manager (WLM). The following ``BatchSettings`` child
+classes are provided per `launcher`:
+
+1. The Slurm `launcher` supports:
+   - :ref:`SbatchSettings<sbatch_api>`
+2. The PBSpro `launcher` supports:
+   - :ref:`QsubBatchSettings<qsub_api>`
+3. The Cobalt `launcher` supports:
+   - :ref:`CobaltBatchSettings<cqsub_api>`
+4. The LSF `launcher` supports:
+   - :ref:`BsubBatchSettings<bsub_api>`
+
+Each above child class allow you to specify the run parameters for a WLM batch job.
+The local `launcher` does **not** support batch jobs.
+
+Once a ``BatchSettings`` object is initialized via the ``Experiment.create_batch_settings()``
+function, you have access to helper functions associated with the object that allow
+configuring the job batch settings. The following chart illustrates helper functions
+provided by the ``SrunSettings`` object. The ``SbatchSettings`` object is used for
+launching batches on Slurm WLM systems.
+
+.. list-table:: SbatchSettings Helper Functions
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - ``SbatchSettings`` func
+     - Example
+     - Description
+   * - ``set_account()``
+     - ``SbatchSettings.set_account(account)``
+     - Set the account for this batch job
+   * - ``set_batch_command()``
+     - ``SbatchSettings.set_batch_command(command)``
+     - Set the command used to launch the batch e.g.
+   * - ``set_nodes()``
+     - ``SbatchSettings.set_nodes(num_nodes)``
+     - Set the number of nodes for this batch job
+   * - ...
+     - ...
+     - ...
+
+In the following :ref:`HPC<HPC>` subsection, we demonstrate configuring a SmartSim entity to launch
+as a batch job on a Slurm system.
+
+===
+HPC
+===
+In the following example, we walk through creating and launching a
+``Model`` entity with ``BatchSettings`` and ``RunSettings`` objects.
+The ``BatchSettings`` object allocates resources for the job while the
+``RunSettings`` object feeds SmartSim the run parameters for the job.
+
+Begin by initializing an ``Experiment`` object
+and specifying the systems WLM to the `launcher` argument.
+In this example, we use the slurm job scheduler:
+
+.. code-block:: python
+
+      exp = Experiment("name-of-experiment", launcher="slurm")
+
+Next, we initialize a simple ``Model`` entity that echos `Hello World` to stdout.
+Begin by initializing a ``RunSettings`` object to configure the run parameters of the ``Model``.
+Note that since we specified the `launcher` as slurm, the slurm specific ``RunSettings`` object
+(``SrunSettings``) will be returned upon initialization:
+
+.. code-block:: python
+
+      srun_settings = exp.create_run_settings("echo", exe_args="Hello World")
+
+Next, initialized a ``BatchSettings`` object. Again, we specified `slurm` as the experiment `launcher`.
```

Comment:
I think you wanted `initialize` without a d

Reply:
Instead of saying "again, ..." which implies one must read all examples (because they have to figure out which one you're referring to obliquely), just state what you did. 

The example should stand alone. 

Consider reusing the same note blocks, if necessary, by extracting them to standalone .rst files and importing, if you find yourself repeating a lot...